www/docs/configuration/providers.md

```diff
@@ -3,13 +3,22 @@ id: providers
 title: Providers
 ---
 
-Authentication Providers in NextAuth.js are services that can be used to sign in (OAuth, Email, etc).
+Authentication Providers in **NextAuth.js** are services that can be used to sign in a user.
 
-## Sign in with OAuth
+There's four ways a user can be signed in:
 
-NextAuth.js is designed to work with any OAuth service, it supports OAuth 1.0, 1.0A and 2.0 and has built-in support for many popular OAuth sign-in services.
+- [Using a built-in OAuth Provider](#oauth-providers) (e.g Github, Twitter, Google, etc...)
+- [Using a custom OAuth Provider](#-using-a-custom-provider)
+- [Using Email](#email-provider)
+- [Using Credentials](#credentials-provider)
 
-### Built-in OAuth providers
+:::note
+NextAuth.js is designed to work with any OAuth service, it supports **OAuth 1.0**, **1.0A** and **2.0** and has built-in support for most popular sign-in services.
+:::
+
+## OAuth Providers
+
+### Available providers
 
 <ul>
 {Object.entries(require("../../providers.json"))
```

Comment:
This list at
https://next-auth-git-docs-1792-providers-params-table-nextauthjs.vercel.app/configuration/providers#available-providers

is getting pretty long. Maybe instead of `ul>li`, we should just have a comma-separated list of the providers similar to how it is done here: https://next-auth-git-docs-1792-providers-params-table-nextauthjs.vercel.app/faq#what-authentication-services-does-nextauthjs-support

Reply:
Nice 💚 , this is how it's gonna look:

<img width="600" alt="Screenshot 2021-05-02 at 21 14 31" src="https://user-images.githubusercontent.com/5938217/116824606-64825900-ab8b-11eb-90da-0de22e6a6870.png">